app/models/user_role.rb

```diff
@@ -40,6 +40,7 @@ class UserRole < ApplicationRecord
 
   EVERYONE_ROLE_ID = -99
   NOBODY_POSITION = -1
+  VALID_COLOR = /\A#?(?:[A-F0-9]{3}){1,2}\z/i # CSS-style hex colors
```

Comment:
Are these "css style" hex colors ... or is it CSS itself that has stolen hex-described color values?!!?